src/components/gui/gui.jsx

```diff
@@ -74,6 +76,8 @@ GUIComponent.propTypes = {
 };
 GUIComponent.defaultProps = {
     basePath: '/',
-    vm: new VM()
+    vm: new VM(),
+    storage: new Storage()
 };
+GUIComponent.defaultProps.vm.attachStorage(GUIComponent.defaultProps.storage);
```

Comment:
@rschamp is this what you had in mind? I feel weird about this line of non-"declarative" code just hanging out here but I don't currently see a better place for it. Open to suggestion of course.